tests/integration/frameworks/wordpress/test_wordpress_transaction_name_hooks_off.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should name WordPress web transaction as an 'Action' with the
+name generated from the template used to generate the page even when
+WordPress hooks are disabled.
+*/
+
+/*SKIPIF*/
+
+/*INI
+newrelic.framework = wordpress
+newrelic.framework.wordpress.hooks = false
+*/
+
+/*ENVIRONMENT
+REQUEST_METHOD=GET
+*/
+
+/*EXPECT_METRICS_EXIST
```

Comment:
Is there an easy way to also check that certain metrics DON'T exist? For example, we want to confirm that `Framework/WordPress/Hook/template_include` isn't here.

Reply:
I think this specific test would require `EXPECT_METRICS` instead for that reason

Reply:
> Is there an easy way to also check that certain metrics DON'T exist? For example, we want to confirm that Framework/WordPress/Hook/template_include isn't here.

I was also pondering on this idea. @ZNeumann Would you prefer `EXPECT_METRICS_DONT_EXIST` or `EXPECT_METRICS_EXIST` with `!` in front of the metric name to negate the check?

Reply:
No strong preference

Reply:
@bduranleau-nr @ZNeumann a2218e0525353e5d52668fddd5744abe282a926f adds `EXPECT_METRICS_DONT_EXIST` and ff796c78abba214eedfbdd8f9c2e9d9f50210910 uses it to enhance test expectation.